Simulation/Pathfinding/LowLevelCBS.cpp

```diff
@@ -2,11 +2,11 @@
 
 Path LowLevelCBS::getIndividualPath(std::shared_ptr<Graph> graph, std::shared_ptr<Agent> agent, std::vector<Constraint> constraints){
     std::shared_ptr<Vertex> u;
-    Action firstAction = agent->getCurrentAction();
+    Action firstAction = agent->getBot()->getCurrentAction();
```

Comment:
Det her kan vi ikke. Eller måske bare et problem ved merge i hvert fald. For vi må ikke have access til controllers fra vores pathfinding for at vi kan teste. 
Så current action bør ligge på agent og ikke controller (Eller i hvert fald som minimum være en property der henter den på agent, så når vi kalder agent->getAgentInfo får den hevet med)

Reply:
Kan ikke rigtigt løses før den anden branch er merged